src/gobgp/Makefile

```diff
@@ -5,10 +5,10 @@ INSTALL := /usr/bin/install
 all: gobgp gobgpd
 
 gobgpd:
-	go get -v github.com/osrg/gobgp/gobgpd
+	/usr/local/go/bin/go get -v github.com/osrg/gobgp/cmd/gobgpd
```

Comment:
So the go path is not set with the new 1.11 version?

Reply:
if you log into slave docker, go is there. Somehow, it didn't work with the makefile.